modules/controllers/error-handling.ts

```diff
@@ -0,0 +1,14 @@
+export async function handleSubgraphErrors<T>(operation: () => Promise<T>): Promise<T | []> {
+    try {
+        return await operation();
+    } catch (error: any) {
+        if (
+            error.message.includes('Too many requests') ||
+            error.message.includes('bad indexers') ||
+            error.message.includes('Bad Gateway')
```

Comment:
I think we should let these errors still throw. We still need to know if crons fail because of it. I dont think its enough to just rely on the subgraph-lag alerts